quarkus/admin/src/main/java/org/apache/polaris/admintool/BootstrapCommand.java

```diff
@@ -30,51 +31,94 @@
     description = "Bootstraps realms and root principal credentials.")
 public class BootstrapCommand extends BaseCommand {
 
-  @CommandLine.Option(
-      names = {"-r", "--realm"},
-      paramLabel = "<realm>",
-      required = true,
-      description = "The name of a realm to bootstrap.")
-  List<String> realms;
+  @CommandLine.ArgGroup(multiplicity = "1")
+  InputOptions inputOptions;
 
-  @CommandLine.Option(
-      names = {"-c", "--credential"},
-      paramLabel = "<realm,clientId,clientSecret>",
-      description =
-          "Root principal credentials to bootstrap. Must be of the form 'realm,clientId,clientSecret'.")
-  List<String> credentials;
+  static class InputOptions {
+
+    @CommandLine.ArgGroup(multiplicity = "1", exclusive = false)
+    StandardInputOptions stdinOptions;
+
+    @CommandLine.ArgGroup(multiplicity = "1")
+    FileInputOptions fileOptions;
+
+    static class StandardInputOptions {
+
+      @CommandLine.Option(
+          names = {"-r", "--realm"},
+          paramLabel = "<realm>",
+          required = true,
+          description = "The name of a realm to bootstrap.")
+      List<String> realms;
```

Comment:
Do we still need this as a separate option? Why not extract after parting the `-c` value?

Reply:
The idea is that users may want to bootstrap a realm without providing credentials (they would be randomly-generated). Should I remove this option? It is true that the file-based approach does not permit randomly-generated credentials.

Reply:
I don't know, honestly... the more it goes, the more I'm in favor of requiring the user to provide all the credentials.

But your suggestion is at least coherent... I'm not opposed to it.

Reply:
In the end, I think it's a minor point compared to the addition of reading secrets from files. We can always refactor that later.